apps/transport/lib/datagouvfr/client/community_resources.ex

```diff
@@ -2,15 +2,17 @@ defmodule Datagouvfr.Client.CommunityResources do
   @moduledoc """
   Helper to access to GET /api/1/community_resources/?dataset=id
   """
-  alias Datagouvfr.Client
+  alias Datagouvfr.Client.HTTPoison, as: Client
```

Comment:
Don't you think that aliasing HTTPoinson with an already existing name can be misleading ?

Reply:
YES I DO!
I will change that to API, is that ok with you?

Reply:
YES I DID!